src/main/java/uk/gov/ons/census/fwmt/jobservice/rest/client/CometRestClient.java

```diff
@@ -73,14 +73,14 @@ private void auth() throws GatewayException {
     }
   }
 
-  public void sendCreateJobRequest(CaseRequest caseRequest, String caseId) throws GatewayException {
+  public <A> void sendCreateJobRequest (A caseRequest, String caseId) throws GatewayException {
```

Comment:
consider renaming this method name. It doesn't only send create jobs now.

Reply:
Name changed to be more generic